packages/types/src/interfaces/grandpa/definitions.ts

```diff
@@ -1,10 +0,0 @@
-export default {
-  rpc: {
-    restartVoter: {
```

Comment:
I think we need this rpc call? else we will get warning for grandpa.rpc.restartVoter does not exist?

Reply:
yeah, wasn't sure how to make it merge with the substrate definitions